Misc/Positions_Area Positions 1 UK Permanent.txt

```diff
@@ -63,14 +88,13 @@ STC-M_CTR:London Control:199.998:SM:L:SCO-M:CTR:-:-::
 
 ;Military
 EGVV_CTR:Swanwick Mil:133.900:VV:M:EGVV:CTR:-:-:6401:6457:N052.44.48.698:W006.55.43.300:N053.12.54.943:W003.16.04.420:N055.58.10.975:W001.19.57.625:N058.29.38.504:W005.05.14.633
+EGVV__CTR:Swanwick Mil:133.900:VV:M:EGVV:CTR:-:-:6401:6457:N052.44.48.698:W006.55.43.300:N053.12.54.943:W003.16.04.420:N055.58.10.975:W001.19.57.625:N058.29.38.504:W005.05.14.633
```

Comment:
I wonder if it's worth explicitly defining the other relief BBX positions if this one is defined

Reply:
Actually we don't need to define the bandboxes, I forgot to take this one out. See resolved convos and comments from Harry